gradle.properties

```diff
@@ -10,11 +10,11 @@ sizeofVersion = 0.3.0
 jaxbVersion = 2.3.1
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.8.0-pre20
+terracottaPlatformVersion = 5.8.0-pre22
 terracottaApisVersion = 1.7.0-pre5
```

Comment:
@mathieucarbou can you also bump the following dependencies:
terracottaApisVersion = 1.7.0-pre6
terracottaCoreVersion = 5.7.0-pre34
terracottaPassthroughTestingVersion = 1.7.0-pre16